deployments/fargate/README.md

```diff
@@ -0,0 +1,143 @@
+# AWS Fargate Deployment
+Familiarity with AWS Fargate (Fargate) is assumed. Consult the 
+[User Guide for AWS Fargate](https://docs.aws.amazon.com/AmazonECS/latest/userguide/what-is-fargate.html)
+for further reading.
+
+Unless stated otherwise, the
+[Splunk OpenTelemetry Connector](https://github.com/signalfx/splunk-otel-collector)
+(Collector) is deployed to an ECS task in a **sidecar** container. We are using the term
+"sidecar" loosely here to mean having the Collector container as a sidecar to multiple or
+all monitored application containers in the task. We do not mean a Collector sidecar container
+per monitored application container as in the strict sense.
```

Comment:
This part seems a bit confusing. I don't understand why is it needed. Specifically this line `Collector container sidecar to multiple or all monitored application containers in the task` doesn't sound right to me.

The collector can be deployed as a sidecar -- another container to one existing task that is being monitored. Or is can be deployed as a standalone task -- like a gateway. I assume this document should specifically describe one way or another. In order to collect metrics from each task, I assume, it should be deployed as sidecar to each container. If so, we need to stick to that approach and document it as prescribed guideline.

Reply:
Ok, I removed the extra confusing bit.